src/orion/plotting/_core.py

```diff
@@ -0,0 +1,85 @@
+import orion.plotting.backend_plotly as backend
+
+
+def regret(experiment, order_by='suggested', verbose_hover=False, **kwargs):
+    """
+    Make a plot to visualize the performance of the hyper-optimization process.
+
+    The x-axis contain the trials and the y-axis their respective best performance.
+
+    Parameters
+    ----------
+        experiment: ExperimentClient, Experiment or ExperimentView
+            The orion object containing the experiment data
+
+        order_by: str
+            Indicates how the trials should be ordered. Acceptable options are below.
+            See attributes of `Trial` for more details.
+
+            * 'suggested': Sort by trial suggested time (default).
+            * 'reserved': Sort by trial reserved time.
+            * 'completed': Sort by trial completed time.
+
+        verbose_hover: bool
+            Indicates whether to display the hyperparameter in hover tooltips. False by default.
```

Comment:
Why not True by default?

Reply:
My rationale was that we want something as streamlined as possible for tooltips, we would see the values of the hyperparameters in a table under the graph (another option that will be added in another PR).

Reply:
Let me know if you prefer it to be true by default

Reply:
I agree with the simplest/leanest default option. I think the utility for the HP values is however large enough to make it a default.